.gitignore

```diff
@@ -67,9 +67,6 @@ content/en/integrations/guide/amazon_cloudformation.md
 !content/en/integrations/snyk.md
 !content/en/integrations/content_security_policy_logs.md
 
-# Ignore the marketplace images that get pulled
-static/images/marketplace
```

Comment:
Might be better to leave this in here for a while until everyone has had a chance to run a clean.

Reply:
Yea this is a good point can we postpone this change @bgdeutsch